### PR TITLE
feat: introduce the feature component

### PR DESCRIPTION
**Pull request by im-klau**
## Name PR according scheme: type of changes (feat/fix/refactor etc.) + description
feat: introduce the feature component

## Jira issue

JIRA issue: [FSU-154](https://klau.atlassian.net/browse/FSU-154)

## Describe your changes
This PR introduces a new `Feature` pattern that consists of icon, heading and description. This element can be used together with the `List` component.

## Checklist before requesting a review

- [x] Checked which branch you would merge.
- [x] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [x] Performed a self-review of code (bugs, clean code rules, etc).
- [x] Checked if added variables have unique names.
- [x] Checked if added code/function is not a duplicate. 
- [x] Tests have been added if possible.
- [x] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: packages/design-system/patterns/feature/Feature.styled.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import styled from "@emotion/styled"
+
+export const Feature = styled.div`
+  display: flex;
+  gap: 32px;
+`
+
+export const Content = styled.div`
+  flex: 1;
```
Comment (AmandaKr) [inline packages/design-system/patterns/feature/Feature.styled.js:9]: There should be 4px gap between heading and description on tablet and mobile I think 🤔

File: packages/design-system/patterns/feature/__stories__/Feature.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { Meta, Canvas, Story, ArgsTable } from "@storybook/addon-docs"
+
+import Feature from "design-system/patterns/feature"
+
+<Meta title="Patterns/Feature" component={Feature} />
+
+# Feature
+
+...
```
Comment (AmandaKr) [inline packages/design-system/patterns/feature/__stories__/Feature.stories.mdx:9]: Description of Feature component is missing here 👀

**Review by paulgrym — APPROVED** [review]
Besides missing gap Amanda spotted I see nothing wrong 😄 Good jos 🚀

File: packages/design-system/patterns/feature/__stories__/Feature.stories.mdx (hunk 2 above)
Comment (im-klau) [reply packages/design-system/patterns/feature/__stories__/Feature.stories.mdx:9]: I totally forgot about this one 🤦‍♀️ thank you!

**Review by AmandaKr — APPROVED** [review]
Perfect ❤️

**Pull request merged**